### PR TITLE
Measure iOS CPU/GPU percentage

### PR DESCRIPTION
**Pull request by liyuqian**
For https://github.com/flutter/flutter/issues/33899

Test added:
- simple_animation_perf_ios

Test modified:
- backdrop_filter_perf_ios__timeline_summary

We'll add the CPU/GPU measurement to more iOS tests
once it's proven to be non-flaky.

### CODE REVIEW

**Review by digiter — COMMENTED** [review]
Where will the result show up? As new graphs on the performance dashboard?

**Review by dnfield — CHANGES_REQUESTED** [review]
We should come up with a better home for the package.

File: dev/devicelab/manifest.yaml [hunk 1]
```diff
@@ -530,6 +530,13 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
 
+  simple_animation_perf_ios:
+    description: >
+      Measure CPU/GPU usage percentages of a simple animation.
+    stage: devicelab_ios
+    flaky: true
```
Comment (goderbauer) [inline dev/devicelab/manifest.yaml]: Please add a comment why this was marked as flaky (e.g. because its new and unproven).

Comment (liyuqian) [reply dev/devicelab/manifest.yaml]: Removed flaky.

**Comment by liyuqian** [comment]
@dnfield @goderbauer @digiter : ready to review again.

**Review by dnfield — APPROVED** [review]
LGTM

Thanks for doing the work to move the package where it belongs - this is much cleaner now!

**Review by digiter — APPROVED** [review]
LGTM

**Pull request merged**